airbase/airtable.py

```diff
@@ -93,33 +93,57 @@ async def _request(self, *args, **kwargs) -> Optional[ClientResponse]:
             else:
                 return res
 
-    def raise_or_log_error(self, error_msg: str) -> None:
+    async def raise_or_log_error(
+        self,
+        res: Optional[ClientResponse],
+        error_msg: str = "",
+    ) -> None:
+        error_msg = await self.get_error_message(res) if res else error_msg
         if self.raise_for_status:
             raise AirbaseException(error_msg)
```

Comment:
can you also pass the res object to `AirbaseException` or perhaps subclass it to `AirbaseResponseException` in the case res is not None ?

Reply:
See latest changes. Added subclass `AirbaseResponseException` and passed on ClientResponse